### ISSUE TITLE
onPassword and onLoadProgress are no longer triggered

### ISSUE THREAD

**Issue by slashvortal** [wojtekmaj/react-pdf#364]
**Description**

The callback is no longer triggered on onPassword:

```
PasswordException {name: "PasswordException", message: "No password given", code: 1}
code: 1
message: "No password given"
name: "PasswordException"
```
Any idea what has been changed recently?

**Environment**

 - Browser Chrome
 - React-PDF version 4.0.2
 - React version 16.6.3
 - Webpack version 4.29.5

**Comment by wojtekmaj** [comment]
Whoops! My bad. Thanks for reporting this!

**Comment by wojtekmaj** [comment]
Fixed in v4.0.5.
